frontend/src/features/admin-form/template/TemplateFormProvider.tsx

```diff
@@ -101,73 +84,14 @@ export const TemplateFormProvider = ({
     [data?.form, data?.spcpSession],
   )
 
-  const { submitEmailModeFormMutation, submitStorageModeFormMutation } =
-    usePreviewFormMutations(formId)
-
-  const handleSubmitForm: SubmitHandler<FormFieldValues> = useCallback(
-    async (formInputs) => {
-      const { form } = data ?? {}
-      if (!form) return
-
-      switch (form.responseMode) {
-        case FormResponseMode.Email:
-          // Using mutateAsync so react-hook-form goes into loading state.
-          return (
-            submitEmailModeFormMutation
-              .mutateAsync(
-                { formFields: form.form_fields, formInputs },
-                {
-                  onSuccess: ({ submissionId }) =>
-                    setSubmissionData({
-                      id: submissionId,
-                      // TODO: Server should return server time so browser time is not used.
-                      timeInEpochMs: Date.now(),
-                    }),
-                },
-              )
-              // Using catch since we are using mutateAsync and react-hook-form will continue bubbling this up.
-              .catch(showErrorToast)
-          )
-        case FormResponseMode.Encrypt:
-          // Using mutateAsync so react-hook-form goes into loading state.
-          return (
-            submitStorageModeFormMutation
-              .mutateAsync(
-                {
-                  formFields: form.form_fields,
-                  formInputs,
-                  publicKey: form.publicKey,
-                },
-                {
-                  onSuccess: ({ submissionId }) =>
-                    setSubmissionData({
-                      id: submissionId,
-                      // TODO: Server should return server time so browser time is not used.
-                      timeInEpochMs: Date.now(),
-                    }),
-                },
-              )
-              // Using catch since we are using mutateAsync and react-hook-form will continue bubbling this up.
-              .catch(showErrorToast)
-          )
-      }
-    },
-    [
-      data,
-      showErrorToast,
-      submitEmailModeFormMutation,
-      submitStorageModeFormMutation,
-    ],
-  )
-
   if (isNotFormId) {
     return <NotFoundErrorPage />
   }
 
   return (
     <PublicFormContext.Provider
       value={{
-        handleSubmitForm,
+        handleSubmitForm: undefined,
         formId,
         error,
         submissionData,
```

Comment:
technically this can be undefined too without the useState initialisation

Reply:
[564ef08](https://github.com/opengovsg/FormSG/pull/5443/commits/564ef08fef4d4645e58581906547cfb923ad9c90)